src/EditorFeatures/Core/Structure/AbstractStructureTaggerProvider.cs

```diff
@@ -39,8 +39,8 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.Structure
     /// editor doesn't know about all the regions in the file, then it wouldn't be able to
     /// persist them to the SUO file to persist this data across sessions.
     /// </summary>
-    internal abstract partial class AbstractStructureTaggerProvider :
-        AsynchronousTaggerProvider<IStructureTag>
+#pragma warning disable CS0618 // Type or member is obsolete
+    internal abstract partial class AbstractStructureTaggerProvider : AsynchronousTaggerProvider<IStructureTag2>
```

Comment:
Looks like IStructureTag2 is no longer experimental, wonder if we could use a newer editor package to avoid the suppression
[https://devdiv.visualstudio.com/DevDiv/_git/VS-Platform?path=/src/Editor/Text/Def/TextUI/Tags/IStructureTag2.cs](https://devdiv.visualstudio.com/DevDiv/_git/VS-Platform?path=/src/Editor/Text/Def/TextUI/Tags/IStructureTag2.cs)